lib/x86simdsort.cpp

```diff
@@ -265,3 +265,47 @@ DISPATCH_KEYVALUE_SORT_FORTYPE(int32_t)
 DISPATCH_KEYVALUE_SORT_FORTYPE(float)
 
 } // namespace x86simdsort
+//
+
+extern "C" {
+XSS_EXPORT_SYMBOL
+void keyvalue_qsort_float_uint32(float *key, uint32_t *val, uint32_t size)
+{
+    x86simdsort::keyvalue_qsort(key, val, size, true);
+}
+XSS_EXPORT_SYMBOL
+void keyvalue_qsort_float_sizet(float *key, size_t *val, size_t size)
```

Comment:
I also think the argument type/name of these being size_t is a little awkward. I think I would probably call them uint64; I guess that might not technically always be the same, if it was running on a 32-bit system. But I presume 64 bit unsigned integer is what is meant here